src/pages/edge/identity-for-edge-network/faq.md

```diff
@@ -113,6 +113,17 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The ECID is reset or cleared in the following cases:
```

Comment:
Can we add a clarifying comment: Identity for Edge Network extension does not reset or clear the ECID. If you are seeing the ECID being reset, identify if it is one of the following cases: